README.md

```diff
@@ -82,9 +60,7 @@ For testing purposes, we are using an internal web server by default at `http://
 The test server simulates some of the W3C services, such as the CSS and HTML validators, or the token authorization checker.
```

Comment:
Again, this relates to my confusion given to the comments above. This seems to relate to the Editor's specification being tests... yet it's part of the testing discussion for Echidna itself.